snippet.py

```diff
@@ -1,7 +1,24 @@
+import os
+from pathlib import Path
+import ssl
+import urllib.request
+
 from pyarrow import flight
 
-client = flight.connect('grpc+tls://flight.spiceai.io')
 
+# Check for gRPC required Google certificate file
+if not Path('/usr', 'share', 'grpc', 'roots.pem').exists():
```

Comment:
Should we add an OS check here? E.g. no point doing this on Windows.

Reply:
Windows seems to have the same issue (according to official gRPC package issue) so this should always return True thus checking the environment variable and avoiding the certificate issue.

So in practice this test should work (should we test it?), I checked the gRPC code and the hard-coded path in indeed platform dependent. If I am reading the code write on windows the environment variable must be used (or in-code argument with the path of the certificate.)

Reply:
Let's support Windows in our SDK. May want to add a note in docs.